src/components/login/signin.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FormField, TextInput, Label, InlineAlert, Button } from 'evergreen-ui';
+import Box from 'ui-box';
+import logo from '../logo/AgroeasyLogo.png';
+
+export default class Signin extends React.Component {
+  render() {
+    return (
+       
+      <FormField display="flex" alignItems="center" justifyContent="center" style={{marginTop:'70px'}}>
+        <div>
+            <span style={{fontWeight: 'bold', fontSize: '20pt', margin: '500px auto'}}>AgroEasy</span>
+            <img src={logo} alt="logo"  style={{width: '60px', height: '30px'}}/>
+            <p style={{fontSize: '20pt'}}>Sign in to your account</p>
+               
+                     
+            <Box>
+                <Box marginBottom={15}>
+                    <Label size={500} display="block" marginBottom={8}>
+                        Email address (required)
```

Comment:
All text should be defined in the constants.js file and then imported and used in the appropriate file